### ISSUE TITLE
Changing a code fails and it must be accepted otherwise nothing else works

### ISSUE THREAD

**Issue by tukanos** [GemTalk/JadeiteForPharo#144]
I'm using GS 3.7.1 and Pharo11 so I'm not sure if the issue was not already fixed.  I have checked the closed issues and it doesn't seems so.

How to get reproduce this bug (actually 2 bugs in one):

1) Have a GS 3.7.1 (RowanV3 extent) vanilla and latest Pharo 11 (with all the required libraries at the versions needed for P11)
2) Connect to GS 
3) Click on the `FileSystemGS` library
4) Click on `FileSystem-Core` package (have the tabs Package, Class and Variable selected)
4.5) Clcik on the `Class` tab to Path class
5) Click on method name e.g. `asFileLocatorOrReference`
6) Make now a change, which will make the background green.
7) Revert the change with shortcut `ctrl + z`, press it once to revert the change

8) 
  a) If you press `ctrl + z` again to undo the next change, which is none, it moves to a method that was selected before.  The selection in the methods stays the same, but the code changes to previously changed method.

  b) The change is reverted, but the background stays green (which it should revert back to yellow).  Now if one tries to change to different method, he will get a message *You have unsaved changes. Discard changes and continue? yes/no*.  If you click on *no* nothing happens and if you try any other action it asks the same question again.  If you want to continue you have to press *yes*.

**Comment by ericwinger** [comment]
Hi tukanos, 
Are you still using jfp [eaad809](https://github.com/GemTalk/JadeiteForPharo/commit/eaad809)?  
If so, that is a really old version so I suspect the problem is already fixed. 
As a precaution, I did double check to see if I could reproduce this issue in the latest code and it didn't reproduce for me. 

fyi - We're in the final testing for GemStone/64 3.7.2. Once that is out (in a couple weeks hopefully), you'll be able to use the latest Jadeite for Pharo code  which is much more stable.

**Comment by tukanos** [comment]
@ericwinger Yes, I'm using it.  I have tried all the latest builds, but still I can't connect with the 3.7.1 libraries to GS 3.7.1.  I guess I will have to wait till GS 3.7.2 is out.

**Comment by tukanos** [comment]
This has been fixed in P12 and GS 3.7.2.
